cmd/mcp-server/eval/scripts/anonymize-outputs.py

```
@@ -0,0 +1,110 @@
+    mapping = {}
+    count = 0
+
+    for scenario in sorted(scenarios):
+        out_dir = blind_dir / scenario
+        out_dir.mkdir(exist_ok=True)
+
+        labels = list(_LABELS[:len(config_dirs)])
+        rng.shuffle(labels)
+
+        scenario_map = {}
+        for config_dir, label in zip(config_dirs, labels):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)